### ISSUE TITLE
instance_size for advance cluster marked as optional in the documentation

### ISSUE THREAD

**Issue by gregoirefra** [mongodb/terraform-provider-mongodbatlas#1311]
### Terraform CLI and Terraform MongoDB Atlas Provider Version

```hcl
Terraform v1.2.6
provider registry.terraform.io/mongodb/mongodbatlas v1.8.1
```

### Terraform Configuration File 

```hcl
resource "mongodbatlas_advanced_cluster" "sandbox" {
  project_id   = var.project_id
  name         = "autoscaling"
  cluster_type = "REPLICASET"
  replication_specs {
    region_configs {
      electable_specs {
        node_count = 3
      }
      auto_scaling {
        disk_gb_enabled           = true
        compute_enabled           = true
        compute_min_instance_size = "M10"
        compute_max_instance_size = "M20"
      }
      provider_name = "AWS"
      priority      = 7
      region_name   = "US_EAST_1"
    }
  }

  provider = mongodbatlas.staging
}

```

### Expected Behavior

Based on the [documentation here](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_cluster#instance_size) the configuration should be valid and not expect an instance size.
### Actual Behavior

Received an error message stating a missing required argument.

```shell
Terraform has been successfully initialized!
╷
│ Error: Missing required argument
│
│   on sandbox.tf line 7, in resource "mongodbatlas_advanced_cluster" "sandbox":
│    7:       electable_specs {
│
│ The argument "instance_size" is required, but no definition was found.
╵

```
### Additional Context

Based on the [code base](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/mongodbatlas/resource_mongodbatlas_advanced_cluster.go) I believe this is simply a documentation error unless we should be able to not specify the instance size when using auto-scaling.

**Comment by andreaangiolillo** [comment]
Hello 👋 
Thanks for opening this issue. The fix will be available in the next release.

Thanks

**Comment by andreaangiolillo** [comment]
Hello,
The fix was released in [v1.11.0.](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest). Feel free to reopen the issue if you still have the same problem with the latest version.

Thanks
